### PR TITLE
Fixes bug where build should be genome_build

### PR DESCRIPTION
**Pull request by moonso**

### CODE REVIEW

**Comment by moonso** [comment]
Could you just do a quick review @northwestwitch ?

File: scout/server/blueprints/variants/controllers.py [hunk 1]
```diff
@@ -275,7 +284,7 @@ def variant_export_lines(store, case_obj, variants_query):
         variant_line.append(variant['chromosome'])
         variant_line.append(position)
         variant_line.append(change)
-        variant_line.append('_'.join([str(position), change]))
+        variant_line.append(str(position)+change)
```
Comment (northwestwitch) [inline scout/server/blueprints/variants/controllers.py]: Why did you change this?
```variant_line.append('_'.join([str(position), change]))```
is correct

Comment (moonso) [reply scout/server/blueprints/variants/controllers.py]: Didn't touch that...

Comment (moonso) [reply scout/server/blueprints/variants/controllers.py]: Fixed now?
